docs/guide/test/quick-start.md

```diff
@@ -1,9 +1,129 @@
-# Coming Soon
+# Running Tests Quick Start
 
-::: info
-This page is Under Construction 🏗️, whilst Moonwall has been built - the docs have not.
+This guide assumes that you've already [installed Moonwall](/guide/intro/getting-started) and you have a basic test case that you want to run. The test case can be a simple transaction that transfers some dust from one test account to another. You can put together a simple test suite here in the [Quick Start Writing Tests guide](/guide/write/quick-start)
 
-Please Stay tuned for more details of this feature!
-:::
+## Getting your Config File Sorted
 
-![underconstruction](/under-construction.png)
+Your `moonwall.config.json` file that lives in the root directory of your project is the key to setting up a sublime testing experience. Go ahead and open it up and let's get started setting it up. In this quick start, we'll set up our Moonwall config file for testing on a local moonbeam node using the [dev foundation](/guide/intro/foundations.html#dev){target=blank}.
+
+### Environment
+
+You'll need to give a name to your environment and you'll need to specify a directory for your test files. The default parameters are as follows:
+
+```json
+"environments": [
+      {
+         "name": "default_env",
+         "multiThreads":false,
+         "testFileDir": [
+            "tests/"
+         ],
+```
+
+For more information about each of the above parameters, please see [environment configuration](/config/environment). 
+
+### Foundation
+
+For this quick start, we've chosen the dev foundation, which utilizes a local development node. We also specify our port information. 
+
+```json
+ "foundation": {
+            "type": "dev",
+            "launchSpec": [{
+              "name": "Moonbeam",
+              "running": true,
+              "newRpcBehaviour": true,
+              "binPath": "tmp/moonbeam",
+              "ports": {"wsPort": 1337}
```

Comment:
newRpcBehaviour deprecates `wsPort`, we should actually instead be specifying `rpcPort`